components/03-modules/slider/_slider.scss

```diff
@@ -0,0 +1,186 @@
+$slider__margin                  : $spacer--medium !default;
+$slider__transition              : $transition-base !default;
+$slider__title-margin            : $spacer--large !default;
+$slider__navigation-margin       : $spacer !default;
+$slider__navigation-icon-spacing : 48px !default;
+$slider__navigation-icon-margin  : $spacer !default;
+$slider__name-margin-right       : $spacer !default;
+$slider__name-font-weight        : $font-weight-base !default;
+$slider__name-font-size          : $font-size-base !default;
+$slider__item-padding-right      : $spacer--medium !default;
+$slider__image-margin            : 0 auto $spacer auto !default;
+$slider__badge-left              : $spacer !default;
+
+$slider__price-old-margin        : $spacer !default;
+$slider__price-font-weight       : $font-weight-bold !default;
+$slider__price-color             : $black !default;
+$slider__price-old-color         : $black !default;
+$slider__price-old-font-weight   : $font-weight-base !default;
+$slider__price-old-font-size     : $font-size-extra-small !default;
+$slider__price-new-color         : $color-primary !default;
+$slider__price-new-font-wieght   : $font-weight-bold !default;
+$slider__price-new-font-size     : $font-size-small !default;
+
+$slider__icon-spacing            : 25px !default;
+$slider__icon-fill               : $color-primary !default;
+$slider__icon-fill-hover         : #007025 !default;
+
+.slider {
+    margin-bottom: $slider__margin;
+    position: relative;
+
+    &__title {
+        margin-bottom: $slider__title-margin;
+        width: 100%;
+    }
+
+    &__handler {
+        overflow: hidden;
+    }
+
+    &__slides {
+        display: flex;
+    }
+
+    &__item {
+        position: relative;
+        padding: 0;
+        min-width: 50%;
+        padding-right: $slider__item-padding-right;
```

Comment:
Doubled padding. Maybe use a shorthand with this item's padding ?

Reply:
done